SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -784,7 +753,6 @@ public void onResponse(RESTResponse response) {
 
                                     if (responseJson.has("location_campaigns")) {
```

Comment:
The location campaigns are saved locally to the sqlite db and we read directly from that. I'll add a note.